docs/integrations/databases/opentelemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+
+Once the logs are configured to be written to a local file, follow the steps below to configure the collection in Sumo Logic.
+
+## Configuring Collection and Installing the Redis App
+
+As part of setting up the collection process and app installation, you can select the App from **App Catalog** and click **Install App**. Follow the steps below:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the option of using an existing Collector.
+:::
+
+If you want to create a new Collector, select the **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent. <br/> <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, we will be configuring the yaml required for Redis Collection.
+
+The log file path configured to capture redis logs must be given here.
+
+The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` file for this information.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
+
+### Step 3: Sending logs to Sumo Logic
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+#### Linux
+
+1. Copy the yaml at **`/etc/otelcol-sumo/conf.d/`** folder in the Redis instance which needs to be monitored.
+2. Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+#### Mac
+
+1. Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2. Restart the otelcol-sumo process using the command `otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"`
+
+Sumo Logic will start receiving data from your host machine when the above command is successfully executed.
+
+This will install the app and dashboards to your Sumo Logic org.
```

Comment:
Is this what you meant in your [comment](https://github.com/SumoLogic/sumologic-documentation/pull/1765#discussion_r1121461962)?

```suggestion
Click **Next** to install the app and dashboards to your Sumo Logic org.
```

Reply:
Yes

Reply:
Thanks for confirming!